### ISSUE TITLE
[entropy_src] V2S Signoff

### ISSUE THREAD

**Issue by andreaskurth** [lowRISC/opentitan#22469]

**Comment by moidx** [comment]
Discussed moving as P1 to M5 due to RTL risk analysis performed during last issue triage.

**Comment by vogelpi** [comment]
## Commits since Earlgrey-PROD.M2

For Commits since Earlgrey-ES tapeout, see https://github.com/lowRISC/opentitan/issues/20973#issuecomment-2026432333 .

`git rev-parse --short HEAD` 3e678c112b

af726e8d9c [entropy_src] Use `es_delayed_enable` for SVAs tracking entropy flow
5b63281fc2 [entropy_src/dv] Avoid Observe FIFO related timeout in max rate test
64c91b860a [entropy_src] Fix prediction for entropy dropping at esfinal FIFO input
e3dea68df8 [entropy_src/dv] Reset CSRNG IF after CS AES Halt IF becoming idle
5784189581 [entropy_src] Add SVA to ensure FIFOs are empty when triggering SHA
-> DV and SVA fixes targeting the max rate test, related to #21855
7f8a9085ab [entropy_src/dv] Close toggle coverage
599e8ade03 [entropy_src/dv] Exclude hard coded counter inputs for toggle coverage
9e50bd3cef [entropy_src/dv] Set thresholds and window sizes appropriately
52275866ea [entropy_src/dv] Remove unimportant cover group bins
-> DV fixes to reach coverage targets for V2/V2S, see #21888
f49daa9d22 [entropy_src/rtl] Delay the main_sm_done signal
-> A fix for an RTL bug discovered when investigating functional coverage issues, see #23773
5caedf3533 [entropy_src/doc] Doc interplay of startup health testing and FW_OV_MODE
cffe272205 [entropy_src/doc] Fix typos
27279ff687 [entropy_src] Locally escalate upon internal FIFO errors and FSM errors
-> Aligning local escalation behavior with EDN and CSRNG
21a6bfdf24 [prim/rtl] Allow for tech-specific impls of `prim_flop_2sync`
-> Not specific to ENTROPY_SRC, related to CDC closure
d9a0305c7a [entropy_src,rtl] Factor sfifo errors into fatal alert
-> Aligning escalation behavior with EDN and CSRNG
6a75417ac9 [entropy_src,dv] Inline entropy_src_assert_if into vseq
ff29c1017f [entropy_src,dv] Remove empty assert_*_alert tasks
-> DV fixes
8295903d7e [bazel,autogen_hjson] Split C and rust header generation
f4c2bb9030 Remove trailing whitespaces
-> Not specific to ENTROPY_SRC
c0fb4c8ce1 [entropy_src] Increase version number, revert verification stage to V1
-> Had to revert to V1 due to functional coverage metrics being below the threshold, see also #21888
92a762764c [entropy_src] Add SEC_CM labels and testplan entries for hardened FIFOs
046e1f035f [entropy_src/doc] Fix typo
3d2bd37379 [entropy_src/doc] Clarify behavior re health test failures and alert
8e89f30018 [entropy_src/doc] Document distribution FIFO
7eff696d84 [entropy_src/doc] Fix documentation of security countermeausures
1254476785 [entropy_src/doc] Clean up documentation of interrupts
e2e0933093 [entropy_src/doc] Clarify that the repetition count tests are continuous
2eb42293ca [entropy_src/doc] Improve structure of theory_of_operation.md
9e8215d95b [entropy_src/doc] Minor corrections to README.md
-> Doc-only changes

## Issues closed since Earlgrey-PROD.M2

For issues closed since Earlgrey-ES tapeout, see https://github.com/lowRISC/opentitan/issues/20973#issuecomment-2026432333 .

- #10071
- #15129
- #16516
- #19789
- #20952
- #20953
- #20976
- #20982
- #21011
- #21855
- #21888
- #22209
- #22223
- #22289
- #22295
- #22385
- #22468
- #22524
- #22732
- #22753
- #23773
- #23832

## Currently open issues

- #2111
- #4139
- #7901
- #10738
- #14363
- #14364
- #14417
- #16276
- #16636
- #17109
- #18109
- #18649
- #18797
- #18836
- #18837
- #18842
- #19392
- #20194
- #21394
- #22241
- #22293
- #22320
- #22323
- #22326
- #22366
- #22467
- #22469
- #22635
- #22668
- #22705
- #22850
- #22851
- #23663
- #23914

**Comment by h-filali** [comment]
In terms of commits I also found:
- 6f274e0332 [entropy_src] Map previously unmapped entropy_src_intr test
- 30d7e787c7 (tag: Earlgrey-PROD-M2-RC5, tag: Earlgrey-PROD-M2) Add the project name to the copyright header
Do we want to add those?
For the open issues I found:
- https://github.com/lowRISC/opentitan/issues/20145
- https://github.com/lowRISC/opentitan/issues/20147
- https://github.com/lowRISC/opentitan/issues/20146
Other than that the issues all fit into this signoff issue.
The coverage report is exactly as you state and has been so for at least a week other than the toggle coverage (which has been fixed in a recent PR) and the entropy_src_intr test which went up from 88% to 100% in the latest regression.
All in all this LGTM.
